SOC/common/simulation.py

```diff
@@ -126,9 +127,10 @@ def run(self, N_iterations: int, filename: str  = None) -> dict:
         self.saved_snapshots.attrs['save_every'] = self.save_every
 
         for i in tqdm.trange(N_iterations):
-            self.drive()
+            self.drive(density*self.L*self.L)
```

Comment:
To chyba nie będzie działać przez to o czym rozmawialiśmy w https://github.com/SocSIM/SocSIM/issues/51#issuecomment-567406853

Może tak? GitHub mi tutaj nie czyta pierwszej linijki jako nadpisującej poprzednią, ale koncept mniej więcej ten:

```suggestion
        for i in tqdm.trange(int(density * self.L * self.L) * N_iterations):
             self.drive()
```

Reply:
Rzeczywiście, zapomniałam o tym kompletnie. Przemyślę to teraz.

Reply:
Nie ma co się zastanawiać, chyba wystarczy przenieść skalowanie do fora i wziu :)

Reply:
poprawiłam tak jak radziłeś